README.md

```diff
@@ -37,6 +37,43 @@ AI tournaments). Changed defaults will be advertised when the match begins.
  - If you would like to develop a bot in Java, please follow [this tutorial](http://sscaitournament.com/index.php?action=tutorial).
  - Want to help improve BWAPI? Fork the [`develop` branch](https://github.com/bwapi/bwapi/tree/develop).
 
+## Quick Start for 4.1.2 {#quick-start-for-4.1.2}
+1. installation
+  1. install Visual Studio 2013
+  2. install StarCraft
+  3. install Brood War
```

Comment:
You can combine steps 2 and 3 and just say "Install Starcraft: Brood War". The original is an implicit dependency for Brood War. Additionally it's combined in the Battle.net installer anyway.